Project.toml

```
@@ -13,9 +12,8 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 ColorVectorSpace = "0.7, 0.8"
-Colors = "0.9.6, 0.10"
 Distances = "0.8.1, 0.8.2"
-ImageCore = "0.8.3, 0.8.4, 0.8.5"
+ImageCore = "0.8.3"
```

```suggestion
ImageCore = "0.8.6"
```

It looks like simply revert the changes are enough, some extra efforts are required to deal with the deprecation warnings:

```julia
┌ Warning: `colordiff(ai::Color, bi::Color, metric::DifferenceMetric)` is deprecated, use `colordiff(ai, bi; metric=metric)` instead.
│   caller = #_colordiff#7 at ciede2000.jl:24 [inlined]
└ @ Core ~/Documents/Julia/ImageDistances.jl/src/ciede2000.jl:24
```

So I'd like to take the easy path, i.e., limiting the lower bound of `ImageCore` here 😂